ui-v2/package.json

```diff
@@ -19,6 +19,7 @@
     "format:css": "prettier --write \"app/styles/**/*.*\"",
     "start": "ember serve --port=${EMBER_SERVE_PORT:-4200} --live-reload-port=${EMBER_LIVE_RELOAD_PORT:-7020}",
     "start:staging": "ember serve --port=${EMBER_SERVE_PORT:-4200} --live-reload-port=${EMBER_LIVE_RELOAD_PORT:-7020} --environment staging",
+    "start:consul": "ember serve --proxy=${CONSUL_HTTP_ADDR:-http://localhost:8500} --port=${EMBER_SERVE_PORT:-4200} --live-reload-port=${EMBER_LIVE_RELOAD_PORT:-7020}",
```

Comment:
ooo is `-val` a way to do default values?

Reply:
Yeah, the `:-` specifically means even if you set the variable to empty (as opposed to not defined) still use the default (as far as I understood)

Some info here:

https://www.tldp.org/LDP/abs/html/parameter-substitution.html